packages/elements/src/datetime-picker/__demo__/index.html

```diff
@@ -468,11 +468,13 @@
             const to = new Date().setSeconds(0, 0);
             const from = new Date(to).setHours(0, 0, 0, 0);
             rangePicker.values = toValues(new Date(from), new Date(to));
+            rangePicker.views = toValues(from, new Date(to));
```

Comment:
There is no need to wrap Date with its constructor here.
```suggestion
            rangePicker.views = toValues(from, to);
```